tardis/plasma/properties/atomic.py

```diff
@@ -787,7 +787,11 @@ def exp1_times_exp(x):
         array_like
             Output array.
         """
-        f = exp1(x) * np.exp(x)
+        x = np.asarray(x)
+        f = np.empty_like(x)
+        small_x = x <= 500
+        f[small_x] = exp1(x[small_x]) * np.exp(x[small_x])
```

Comment:
What warning does this fix?

Reply:
RuntimeWarning: overflow encountered in exp
    f = exp1(x) * np.exp(x)